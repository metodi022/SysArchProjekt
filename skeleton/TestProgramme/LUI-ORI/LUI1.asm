lui $1, 65535   # set $1 to ffff0000
