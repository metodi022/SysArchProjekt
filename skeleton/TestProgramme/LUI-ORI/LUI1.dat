0x3c01ffff
