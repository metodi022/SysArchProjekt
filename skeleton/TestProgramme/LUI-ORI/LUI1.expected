@1 ffff0000
